app.cfg.example

```diff
@@ -134,3 +134,24 @@ poll_interval = 60
 
 # full path to the command for manipulating existing jobs
 scontrol_command = /usr/bin/scontrol
+
+
+[submitted_job_comments]
```

Comment:
add a comment above here to indicate that some of these comments should not be changed because we have regular expression patterns in the code that match them